conf/local.yml

```diff
@@ -1,9 +1,9 @@
 ---
 
-host: 127.0.0.1
+host: 0.0.0.0
```

Comment:
why default to opening on all interfaces?

Reply:
This is debug stuff I was doing that accidentally got submitted in the update somehow.